lib/diagnostic.rb

```diff
@@ -89,22 +99,21 @@ def method_name
 
 # Predict what value will be returned. Write your response as a Ruby string.
 
-Diagnostic.flow_control = 'your response here'
+Diagnostic.flow_control = 'The Dark Knight'
 
 # In a Ruby string, write what keyword you use for "else if" clauses in Ruby.
 
-Diagnostic.else_if = 'your response here'
+Diagnostic.else_if = 'elsif'
 
 # Looping in Ruby
 #
 # What method can you call on an array in order to iterate over it in Ruby?
 # Write your answer as a symbol.
 
-Diagnostic.array_iteration = :example
+Diagnostic.array_iteration = :each
 
 # Returns in Ruby
 #
 # Does ruby require an explicit return from methods? Answer true or false, and
 # replace nil.
-
-Diagnostic.returns = nil
+Diagnostic.returns = 'no it returns the last line'
```

Comment:
I'll give you this but please read closely:

"Answer true or false, and
  # replace nil."